cellprofiler/modules/thresholding.py

```diff
@@ -0,0 +1,231 @@
+# coding=utf-8
+
+"""
+
+Thresholding
+
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+import numpy
+import skimage.exposure
+import skimage.filters
+import skimage.morphology
+
+
+class Thresholding(cellprofiler.module.Module):
+    category = "Volumetric"
+    module_name = "Thresholding"
+    variable_revision_number = 1
+
+    def create_settings(self):
+        self.x_name = cellprofiler.setting.ImageNameSubscriber(
+            u"Input"
+        )
+
+        self.y_name = cellprofiler.setting.ImageNameProvider(
+            u"Output",
+            u"OutputImage"
```

Comment:
I like “Thresholding” because it’s consistent across image processing modules (i.e. the default output image name can be the name of the module).